src/System.Globalization/tests/NumberFormatInfo/NumberFormatInfoCurrentInfo.cs

```diff
@@ -20,46 +21,35 @@ public static IEnumerable<object[]> CurrentInfo_CustomCulture_TestData()
         [MemberData(nameof(CurrentInfo_CustomCulture_TestData))]
         public void CurrentInfo_CustomCulture(CultureInfo newCurrentCulture)
         {
-            CultureInfo oldCurrentCulture = CultureInfo.CurrentCulture;
-            try
+            RemoteInvoke((cultureCode) =>
             {
-                CultureInfo.CurrentCulture = newCurrentCulture;
-                Assert.Same(newCurrentCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
-            }
-            finally
-            {
-                CultureInfo.CurrentCulture = oldCurrentCulture;
-            }
+                CultureInfo newCulture = new CultureInfo(cultureCode);
+                CultureInfo.CurrentCulture = newCulture;
+                Assert.Same(newCulture.NumberFormat, NumberFormatInfo.CurrentInfo);
+                return SuccessExitCode;
+            }, newCurrentCulture.TwoLetterISOLanguageName).Dispose();
```

Comment:
I assume your local machine is Windows 10, not Windows 8?  I expect the problem is that you're trying to construct a CultureInfo with an unrecognized culture code (the logs from the run have been deleted, but I'd have expected the failure to be something like a CultureNotFoundException for the invariant input).  Why are you using TwoLetterISOLanguageName rather than Name?  That'll produce "iv" for InvariantCulture, which won't be recognized as a valid input to the ctor, and that'll cause an exception pre-Windows 10.  TwoLetterISOLanguageName would also make the two "en" and "en-US" inputs identical.